sdk/storage/azure-storage-common/CMakeLists.txt

```diff
@@ -73,10 +73,9 @@ target_include_directories(
     PUBLIC
       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
       $<INSTALL_INTERFACE:include>
-      ${azure-core-cpp_INCLUDE_DIRS}
```

Comment:
Don't remove.

Reply:
duplicate, discuss in another conversation